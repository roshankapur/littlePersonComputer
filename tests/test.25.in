1020
2020
4210
2013
3320
2113
2020
3112
2120
4002
1113
4300
1
1
-99999
8