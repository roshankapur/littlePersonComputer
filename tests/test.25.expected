***Start LPC Execution***
***FATAL ERROR - Accumulator Overflow***
***lpc execution abnormally terminated***

Registers: 
accumulator +6720
program counter 00004
op-code 00033
operand 00020

      0    1    2    3    4    5    6    7    8    9 
 0 1020 2020 4210 2013 3320 2113 2020 3112 2120 4002 
10 1113 4300 0001 6720 0000 0000 0000 0000 0000 0000 
20 0003 0000 0000 0000 0000 0000 0000 0000 0000 0000 
30 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
40 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
50 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
60 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
70 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
80 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
90 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
***End LPC Execution***
